docs/PT_BR/HPCCSpark/SparkHPCC.xml

```diff
@@ -102,16 +104,7 @@
       HPCC.</para>
 
       <para>O repositório git inclui dois exemplos na pasta
-      Examples/src/main/scala folder. Os exemplos
-      (<emphasis>org.hpccsystems.spark_examples.Dataframe_Iris_LR</emphasis> e
-      <emphasis>org.hpccsystems.spark_examples.Iris_LR</emphasis>) são Scala
-      Objects com uma função main(). Ambos os exemplos usam o dataset clássico
-      da Iris. O dataset pode ser obtido de uma variedade de fontes, incluindo
-      o repositório HPCC-Systems/ecl-ml. IrisDs.ecl (pode ser encontrado na
-      pasta ML/Tests/Explanatory: <ulink
-      url="https://github.com/hpcc-systems/Spark-HPCC/blob/master/Examples/src/main/ecl/IrisDS.ecl">https://github.com/hpcc-systems/Spark-HPCC/blob/master/Examples/src/main/ecl/IrisDS.ecl</ulink>)
-      pode ser executado para gerar o dataset Iris no HPCC. Um passo a passo
-      dos exemplos é fornecido na seção Exemplos.</para>
+      Examples/src/main/scala folder. Os exemplos</para>
```

Comment:
this doesn't match what was done in https://github.com/hpcc-systems/HPCC-Platform/pull/19449/files

I think line 83 (after: "você pode encontrar o código-fonte e exemplos.")-107 should be removed as well